### PR TITLE
chore(deps): update jest monorepo to v29 (major)

### PR DESCRIPTION
**Pull request by winniehell**
updates Jest from 27 to 29

supersedes https://github.com/posva/vuex-mock-store/pull/186

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,7 +63,8 @@
       "<rootDir>/tests/*.js",
       "<rootDir>/tests/.*.js",
       "<rootDir>/tests/*/*.js"
-    ]
+    ],
+    "testEnvironment": "jsdom"
```
Comment (winniehell) [inline package.json:67]: this was the default in previous versions and is necessary for @vue/test-utils

**Comment by winniehell** [comment]
CircleCI didn't run here and I'm not sure why. Maybe that will be resolved after merging https://github.com/posva/vuex-mock-store/pull/198.

**Comment by winniehell** [comment]
@posva this is now rebased and ready for review

**Review by posva — APPROVED** [review]
Thanks, you rock!

**Pull request merged**